compiler/test/suites/basic_functionality.re

```diff
@@ -164,6 +164,11 @@ describe("basic functionality", ({test, testSkip}) => {
   assertSnapshot("if_one_sided4", "let mut x = 1; if (3 < 4) x = 2; x");
   assertSnapshot("if_one_sided5", "let mut x = 1; if (3 < 4) x = 2 + 3");
   assertSnapshot("if_one_sided6", "let mut x = 1; if (3 < 4) x = 2 + 3; x");
+  assertCompileError(
+    "if_empty_block",
+    "if (false) 1 else {}",
+    "Syntax error",
+  );
```

Comment:
I added this test because I was like "was an empty block after `else` ever valid syntax?" And the answer is nope, it errors with "Expected an expression or a record field."